scripts/wasmd/start.sh

```diff
@@ -26,7 +26,7 @@ docker run --rm \
   --name "$CONTAINER_NAME" \
   -p "$TENDERMINT_PORT_HOST":"$TENDERMINT_PORT_GUEST" \
   -p "$LCD_API_PORT_HOST":"$LCD_API_PORT_GUEST" \
-  --mount type=bind,source="$SCRIPT_DIR/template",target=/template \
+  --mount type=bind,source="$SCRIPT_DIR/template_v50",target=/template \
```

Comment:
We can overrise template since we have no way to switch between different wasmd version anyways